renderer/src/common/components/settings/tabs/__tests__/version-tab.test.tsx

```diff
@@ -5,6 +5,13 @@ import { PromptProvider } from '@/common/contexts/prompt/provider'
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import type { AppVersionInfo } from '@/common/hooks/use-app-version'
 import userEvent from '@testing-library/user-event'
+import { toast } from 'sonner'
+
+vi.mock('sonner', () => ({
```

Comment:
probably not necessary to do a one-off mock here, i think this should already be mocked globally

Reply:
(I guess that the mock could be importable, in order to be able to verify what it was called with)

Reply:
probably yes, I remember that we talked alrady of this 😆 let me remove it